test/common/json/test_data/generate_test_data.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env python
```

Comment:
This file is going to get huge and hard to maintain. At minimum, can we add comments for folks on best practices on adding a new schema test (best would be comments in here and some type of README.me). Better I think would be to break this program into multiple Python files.